### DISCUSSION TITLE
Configuration.yaml

### DISCUSSION THREAD

**Discussion by Rastablastaman** [indomus/forum#2864] [HUB: Home Assistant]
Buongiorno a tutti, 
  Sono Marco e vorrei iniziare a usare Home Assistant e ho cominciato ad installarlo seguendo la guida di Indomus.it, arrivando fino ad "aprire il tunnel". a questo punto, la guida suggerisce di riconfigurare Home Assistant, aggiunngendo il blocco http nel file configuration.yaml. 
provo ad aprire il file configuration.yaml, ma mi viene evidenziato che il file non è riscrivibile, quindi non midificabile. come faccio a modificare configuration.yaml?
Probabilmente è una problematica da principianti, ma seguendo la guidi ho provato e riprovato...ma niente, non riesco  a creare un file modificabile e quindi non riesco ad andare avanti. Ho installato anche visual studio core, raggiungo la cartella in cui si trova il file, ma ovviamente quando provo a salvare, mi dice che non si può salvare. 

probabilmente ho sbagliato qualche passaggio nelle installazioni....ho provato ad installare Home Assistant Core su Docker, ma non riesco proprio a capire dove sbaglio!

c'è qualcuno che riesce ad aiutarmi? so che sarà una stupiddaggine per i più...ma proprio non so da dove partire.

vi ringrazio anticipatamente

ciao 

Marco

**Comment by snakuzzo** [comment]
Tema già trattato, pochissimi giorni fa.
Prima di chiedere fate una ricerca, altrimenti si generano discussioni tutte uguali.
Leggi [QUI](https://github.com/indomus/forum/discussions/2847) e troverai la soluzione

**Reply by Rastablastaman** [reply]
Grazie mille! Gentilissimo!

**Reply by Rastablastaman** [reply]
Ho letto il tema che mi hai suggerito...ma purtroppo capito pochissimo....sono veramente "ignorante"....non riesco proprio a capire quli comandi dare per poter effettuare quello che hai suggerito!

**Reply by Scialla** [reply]
Devi cambiare l'owner, il proprietario della cartella e dei file, in modo ricorsivo (anche alle sottocartelle) con:
`sudo chown -R <tuoutente>:<tuogruppo> <directory>`
